spec/helpers/text_format_helper_spec.rb

```diff
@@ -30,4 +30,18 @@
       end
     end
   end
+
+  describe "#safe_html_format" do
+    subject { safe_html_format html }
+
+    context "with allowed HTML" do
+      let(:html) { "<p><strong>hello</strong> <a href=\"http://test.com\">world</a></p><ul><li>test</li></ul>" }
+      it { is_expected.to eql html }
+    end
+
+    context "with disallowed HTML" do
```

Comment:
Can we add a test in here for malicicious attributes, eg `<a onclick` - I can see the code handles this but would be good to have a test for that behaviour?

Reply:
Yeah good call, I'll do that on both PRs